packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -9,40 +9,51 @@ const trace = makeTracer('NewAuction', true);
 /**
  * @typedef {PromiseSpaceOf<{
  *   auctionUpgradeNewInstance: Instance;
+ *   newContractGovBundleId: string;
```

Comment:
And this promise space key is shared across the whole bootstrap vat, right?

At some point, it belongs in [ChainBootstrapSpaceT](https://github.com/Agoric/agoric-sdk/blob/9ede17dae6db936ce5efe3e3c37d739b9d5f347e/packages/vats/src/core/types-ambient.d.ts#L336). I wonder when we should do that. hm.